roles/template_service_broker/vars/openshift-enterprise.yml

```diff
@@ -1,4 +1,4 @@
 ---
 __template_service_broker_prefix: "registry.access.redhat.com/openshift3/"
-__template_service_broker_version: "v3.7"
+__template_service_broker_version: "{{ openshift_template_service_broker_image_tag | default(openshift_release) }}"
```

Comment:
These should likely just be `{{ openshift_release}}` since we have logic to overwrite this default in https://github.com/openshift/openshift-ansible/blob/master/roles/template_service_broker/tasks/install.yml#L9